content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,357 @@ schema: olm.channel
 
 Here, a channel is generated for each template channel which differs by minor version, and each channel has a `replaces` edge from the predecessor channel to the next-lesser minor bundle version. Please note that at no time do we transgress across major-version boundaries with the channels, to be consistent with [the semver convention](https://semver.org/) for major versions, where the purpose is to make incompatible API changes.
 
+## Composite Template
+You can use a `composite template` to generate a valid FBC for multiple catalogs at a time. A `composite template` is an input schema that accepts a configuration schema. Catalog maintainers can use a `composite template` to build multiple versions of a catalog while maintaining fewer files. Operator authors can use a `composite template` to contribute their operator to multiple catalogs.
+
+### Usage
+```sh
+opm alpha render-template composite [flags] <filename>
+```
+
+| Flag                | Description                                                                            |
+| ------------------- | -------------------------------------------------------------------------------------- |
+| -h, --help          | prints help/usage information                                                          |
+| -o, --output <type> | the output format, can be `yaml` or `json`                                             |
+| -c, --composite-config <file> | File to use as the composite configuration file (default "catalog/config.yaml") |
```

Comment:
@everettraven we need a PR to solve:
- changing (code, and now doc) references from 'catalog/config.yaml' to 'contributions.yaml'
- removing references to container tools (my bad!) (code, and now doc)

I can do it after the current queue of reviews, or you can do it and add to the queue. ;)

Reply:
Just to make sure I am following - is this a follow-up that needs to be done in both a new code change and docs change PR?

Reply:
2 independent PRs, since docs & code are discrete.  I was just making sure I captured the thesis so it didn't get lost.  Not scope for this PR, IMHO.